lib/secret_config/providers/ssm.rb

```diff
@@ -10,7 +10,13 @@ module Providers
     class Ssm < Provider
       attr_reader :client, :key_id, :retry_count, :retry_max_ms, :logger
 
-      def initialize(key_id: ENV["AWS_ACCESS_KEY_ID"], key_alias: ENV["AWS_ACCESS_KEY_ALIAS"], retry_count: 10, retry_max_ms: 3_000)
+      def initialize(
+        key_id: ENV["SECRET_CONFIG_KEY_ID"],
+        key_alias: ENV["SECRET_CONFIG_KEY_ALIAS"],
+        retry_count: 10,
+        retry_max_ms: 3_000,
+        **args
+      )
         @key_id       =
```

Comment:
Changed env variables to better align with a common naming guidelines for gem (libraries) configuration.